framework/decode/dx12_resource_value_mapper.cpp

```diff
@@ -1226,11 +1251,30 @@ void Dx12ResourceValueMapper::MapValue(const ResourceValueInfo& value_info,
         GFXRECON_CHECK_CONVERSION_DATA_LOSS(size_t, value_info.size);
         util::platform::MemoryCopy(&desc, sizeof(desc), desc_data_ptr, static_cast<size_t>(value_info.size));
         GetDispatchRaysResourceValues(indirect_values_map, value_info.state_object, desc);
+        return true;
+    }
+    else if (value_info.type == ResourceValueType::kExecuteIndirectCountBuffer)
+    {
+        // Read command counts while CountBuffer contents is still visible in temp_resource_data
+        uint32_t command_count = 0;
+        util::platform::MemoryCopy(&command_count,
+                                   sizeof(command_count),
+                                   temp_resource_data.data() + value_info.offset,
+                                   sizeof(command_count));
+
+        // Insert new ArgumentBuffer RV infos, which will queue them for translation.
+        GetExecuteIndirectResourceValues(indirect_values_map[value_info.arg_buffer_extra_info.argument_buffer],
```

Comment:
If `command_count == 0`, this call to `GetExecuteIndirectResourceValues` can be skipped. This should avoid triggering the warning `Did not find resource value info for the given object.` in `Dx12ResourceValueMapper::MapResources`

Reply:
`GetExecuteIndirectResourceValues` will go through each argument by the command_count, if `command_count == 0`, the functions will return immediately, won't add any value to the map.

> If `command_count == 0`, this call to `GetExecuteIndirectResourceValues` can be skipped. This should avoid triggering the warning `Did not find resource value info for the given object.` in `Dx12ResourceValueMapper::MapResources`

Reply:
The first argument `indirect_values_map[value_info.arg_buffer_extra_info.argument_buffer]` will add an entry to `indirect_values_map` for `value_info.arg_buffer_extra_info.argument_buffer` even though that entry won't contain any values.